### ISSUE TITLE
Chnage all the get requests to post request

### ISSUE THREAD

**Issue by M-hell** [M-hell/Promptopia#1]
In app/api check all the routes of backend that have http request type get change it to post, then go to the frontend routes and check where those get requests has been called change those post request as well by changing the method to post. 

The codebase is really small so it should'nt be a problem to find errors.

**Comment by sujalcharati** [comment]
@M-hell  can i be assigned under gssoc and hacktoberfest ?

**Comment by M-hell** [comment]
@sujalcharati  yeah ofc just send me a PR after solving the problem i will merge it after inspecting

**Comment by sujalcharati** [comment]
ok,
just assign me so that others dont waste their time here and can it be label it under  gssoc label?

**Comment by M-hell** [comment]
@sujalcharati sorry but i amnot a maintainer of gssoc

**Comment by sujalcharati** [comment]
@M-hell  never mind
